WordPress/Classes/ViewRelated/Post/Revisions/RevisionsTableViewController.swift

```diff
@@ -125,6 +142,13 @@ extension RevisionsTableViewController: WPTableViewHandlerDelegate {
         configureCell(cell, at: indexPath)
         return cell
     }
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
```

Comment:
I wonder if this should be animated or not.  It looks like when animated is `true` the deselection animation doesn't fade in the diff icons.  They "pop" in at the very end.  It would be good to get @iamthomasbishop's to take a look.
![revision-cell-animation 2018-11-09 17_29_15](https://user-images.githubusercontent.com/1435271/48293508-0e624e00-e445-11e8-8b26-35a04dbf7664.gif)

Reply:
@aerych @iamthomasbishop I pushed a fix. It seems like `UITableViewCell` changes the background colour of all the sub views when the cell is selected or highlighted.
To fix it I set the images every time is selected.